### PR TITLE
Add boilerplate v0.5.2

### PR DESCRIPTION
**Pull request by cblecker**
ref https://github.com/openshift/boilerplate/pull/158

### CODE REVIEW

File: clusters/app.ci/supplemental-ci-images/boilerplate/boilerplate.yaml [hunk 1]
```diff
@@ -43,3 +43,9 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/app-sre/boilerplate:image-v0.5.1
+  - annotations:
+      openshift.io/imported-from: quay.io/app-sre/boilerplate:image-v0.5.2
+    name: image-v0.5.1
```
Comment (2uasimojo) [inline clusters/app.ci/supplemental-ci-images/boilerplate/boilerplate.yaml:48]:
```suggestion
    name: image-v0.5.2
```

**Review by 2uasimojo — CHANGES_REQUESTED** [review]
We'll also need to add the image to our [DPTP-1640 workaround](https://github.com/openshift/release/blob/1a4b43ee6ec0725d9d474d89ecdbe40c6c0f1f10/ci-operator/config/openshift/boilerplate/openshift-boilerplate-master.yaml). Note that that depends on this change, so either we do it in this PR and we'll have to merge despite failed rehearsals; or we do it in a separate PR.

**Comment by cblecker** [comment]
@2uasimojo updated

**Comment by 2uasimojo** [comment]
The [boilerplate side](https://github.com/openshift/boilerplate/pull/158) is merged.

This one is expected to fail rehearsals due to chicken/egg problem: the supplemental-ci-images change is necessary to produce the image referenced by the ci-operator config change. We could split those into two PRs, or we could just rely on visual inspection and merge this all-in-one. I'm going with the latter.

/lgtm
/approve

**Pull request merged**